package.json

```diff
@@ -241,7 +241,7 @@
     "prettier": "^2.6.2",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-query": "^3.39.1",
+    "react-query": "^4.0.0",
```

Comment:
Actually, we should use `@tanstack/query-core`.
Are you available to work on it?
(Anyway, I'll cut a release before that, with RQ v3.)

Reply:
Also migrated to `@tanstack/query-core`.
I haven't focus on this library lately. I thought it was still `react-query` and It's my fault😂